multinet/api/models/workspace.py

```diff
@@ -81,6 +81,32 @@ def get_user_permission(self, user: User) -> Optional[WorkspaceRole]:
         """Get the WorkspaceRole for a given user on this workspace."""
         return WorkspaceRole.objects.filter(workspace=self.pk, user=user.pk).first()
 
+    def get_user_permission_level(self, user: User) -> Optional[str]:
```

Comment:
Sorry to complicate things more, but looking back at it, I think a function name of `get_user_permission_label` would be the best of both worlds. This would imply that it's related to `get_user_permission`, but also indicates that it's different in a way, and would align it more with reality, since `WorkspaceRoleChoice(choice).label` is what is returned (except for `owner`). On the surface it's unclear the difference between `get_user_permission` and `get_user_permisson_level`, as they seem to describe the same thing.
```suggestion
    def get_user_permission_label(self, user: User) -> Optional[str]:
```